src/lua/auto/coroutines.lua

```diff
@@ -1,14 +1,16 @@
 require("utils.path")
+require("utils.teleoproutine")
+
 local taxiPath = orientPath(readPath("taxi"))
 local autoPath1 = orientPath(readPath("auto1"))
 local autoPath2 = orientPath(readPath("auto2"))
 local pprint = require("utils.pprint")
 
-doNothingAuto = FancyCoroutine:new(function()
+doNothingAuto = TeleopRoutine:new(function()
```

Comment:
This is all sort of jank, still need to update this and test it in the simulator.